src/measure_theory/decomposition/jordan.lean

```diff
@@ -64,9 +64,20 @@ variable (j : jordan_decomposition α)
 instance : inhabited (jordan_decomposition α) :=
 { default := ⟨0, 0, mutually_singular.zero⟩ }
 
+instance : has_neg (jordan_decomposition α) :=
+{ neg := λ j, ⟨j.ν, j.μ, j.mutually_singular.symm⟩ }
+
```

Comment:
I guess you can do the same for `has_zero` too? That's probably worth having lemmas about.